src/client/public/index.script.ts

```diff
@@ -15,29 +15,27 @@ let { config } = PLAYGROUND;
     chooseHost.textContent = strings.HtmlPageStrings.chooseYourHost;
     chooseHost.style.visibility = 'visible';
 
-    let ribbons = document.getElementsByClassName('ribbon');
-    for (let i = 0; i < ribbons.length; i++) {
-        let ribbon = ribbons[i] as HTMLElement;
-        switch (environment.current.config.name) {
-            case config['insiders'].name:
-                ribbon.textContent = 'Beta';
-                ribbon.style.background = 'red';
-                break;
-            case config['edge'].name:
-                ribbon.textContent = 'Alpha';
-                ribbon.style.background = 'blue';
-                break;
-            case config['local'].name:
-                ribbon.textContent = config['local'].editorUrl;
-                ribbon.style.background = 'green';
-                break;
-            default:
-                break;
-        }
-
-        if (environment.current.config.name !== config['production'].name) {
-            ribbon.style.visibility = 'visible';
-        }
+    let ribbon = document.getElementById('ribbon');
+    switch (environment.current.config.name) {
+        // Utilize Office pallet colors for ribbons
+        case config['insiders'].name:
```

Comment:
The compiler complains if I use dot notation. Not sure why, but intellisense doesn't recognize `config.insiders`